src/Simulation/Common/Simulators.Type1.Dev.props

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  
+  <PropertyGroup>
+    <DocumentationFile>bin\$(BuildConfiguration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+    <EnlistmentRoot>$([MSBuild]::NormalizeDirectory($(MSBuildThisFileDirectory)..\..\..\))</EnlistmentRoot>
+    <NativeBuildPath>$([MSBuild]::NormalizePath($(EnlistmentRoot)src/Simulation/Native/build/))</NativeBuildPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(QsimDll)' == ''">
+    <QsimDllMac>$([MSBuild]::NormalizePath($(NativeBuildPath)/libMicrosoft.Quantum.Simulator.Runtime.dylib))</QsimDllMac>
+    <QsimDllLinux>$([MSBuild]::NormalizePath($(NativeBuildPath)/libMicrosoft.Quantum.Simulator.Runtime.so))</QsimDllLinux>
+    <QsimDllWindowsRelease>$([MSBuild]::NormalizePath($(NativeBuildPath)/Release/Microsoft.Quantum.Simulator.Runtime.dll))</QsimDllWindowsRelease>
+    <QsimDllWindowsDebug>$([MSBuild]::NormalizePath($(NativeBuildPath)/Debug/Microsoft.Quantum.Simulator.Runtime.dll))</QsimDllWindowsDebug>
+    <QSimDll Condition="$([MSBuild]::IsOsPlatform('OSX'))">$(QsimDllMac)</QSimDll>
+    <QSimDll Condition="$([MSBuild]::IsOsPlatform('Linux'))">$(QsimDllLinux)</QSimDll>
+    <QSimDll Condition="$([MSBuild]::IsOsPlatform('Windows'))">$(QsimDllWindowsRelease)</QSimDll>
+    <QSimDll Condition="$([MSBuild]::IsOsPlatform('Windows')) And Exists('$(QsimDllWindowsDebug)')">$(QsimDllWindowsDebug)</QSimDll>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(EnlistmentRoot)src\Simulation\Core\Microsoft.Quantum.Runtime.Core.csproj" />
+    <ProjectReference Include="$(EnlistmentRoot)src\Simulation\Type1Core\Microsoft.Quantum.Type1.Core.csproj" />
+    <ProjectReference Include="$(EnlistmentRoot)src\Simulation\Common\Microsoft.Quantum.Simulation.Common.csproj" IncludeInSimulatorPackage="true" />
+    <ProjectReference Include="$(EnlistmentRoot)src\Simulation\Simulators.Type1\Microsoft.Quantum.Simulators.Type1.csproj" Condition=" '$([MSBuild]::NormalizePath($(EnlistmentRoot)src\Simulation\Simulators.Type1))' != '$(MSBuildProjectDirectory)'" IncludeInSimulatorPackage="true" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="$(QSimDll)" >
+      <Link>Microsoft.Quantum.Simulator.Runtime.dll</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Visible>false</Visible>
+    </None>
+  </ItemGroup>
+  
+  <Target Name="BeforeCsharpCompile">
+    <ItemGroup>
+      <Compile Include="$(GeneratedFilesOutputPath)**/*.g.cs" Exclude="@(Compile)" AutoGen="true" />
+    </ItemGroup>
+  </Target>
+
+</Project>
```

Comment:
This looks like it's identical to Simulators.Type2.Dev.props except for the project reference. Is it possible to factor these two files out into a common props file?

Reply:
I did try a variation like that, where all the simulators shared a single .props file that differentiated between which packages to load via conditionals. It does work, but the trade off for the reduced duplication is some extra complexity in seeing how files are loaded where. I ended up going back to this version in order to be more explicit. I could probably be convinced to address this in a follow up though ;)

Reply:
I wasn't thinking of conditionals, but a shared props file that has everything except the type1/2 project references. Then the type1/2 projects would be something like

```xml
<Project>
  <Import Project="Common.props" />
  <ItemGroup>
    <ProjectReference Include="$(EnlistmentRoot)src\Simulation\Type2Core\Microsoft.Quantum.Type2.Core.csproj" />
    <ProjectReference Include="$(EnlistmentRoot)src\Simulation\Simulators.Type2\Microsoft.Quantum.Simulators.Type2.csproj" Condition=" '$([MSBuild]::NormalizePath($(EnlistmentRoot)src\Simulation\Simulators.Type2))' != '$(MSBuildProjectDirectory)'" IncludeInSimulatorPackage="true" />
  </ItemGroup>
</Project>
```

Does that not work?
    <ProjectReference Include="$(EnlistmentRoot)src\Simulation\Type2Core\Microsoft.Quantum.Type2.Core.csproj" />

Reply:
Oh, I see what you are suggesting... yeah, that does seem like it could work! I'll try pulling all the type specific stuff into the respective projects and then using the props file for only the things they have in common. Good thinking!

Reply:
Finally had the chance to clean this up, and it looks much better this way. Thanks for the suggestion!